parquet/src/test/scala/magnolify/parquet/ParquetTypeSuite.scala

```diff
@@ -44,7 +44,11 @@ class ParquetTypeSuite extends MagnolifySuite {
     t: ParquetType[T],
     eq: Eq[T]
   ): Unit = {
+    // Ensure serializable even after evaluation of `schema`
+    val parquetSchema = t.schema
+    val avroSchema = t.avroSchema
     val tpe = ensureSerializable(t)
+
```

Comment:
Does it reproduce here - the test fail without the fix?

Reply:
yes, the test failed before my code change :)